pkgs/by-name/sl/slimserver/package.nix

```diff
@@ -51,6 +51,7 @@ perlPackages.buildPerlPackage rec {
       ArchiveZip
       AsyncUtil
       AudioScan
+      CarpAssert
```

Comment:
Not sure how this was missed before, but since they updated it in this release I pulled it in.